docs/reference/field-ref.asciidoc

```diff
@@ -2,32 +2,45 @@
 [role="xpack"]
 == Elastic Security ECS field reference
 
-This section lists ECS fields Elastic Security uses to display data.
+This section lists ECS fields used by {es-sec} to provide an optimal SIEM and/or security analytics experience to users.  These fields are used to display data, provide rule previews, enable detection by prebuilt detection rules, provide context during rule triage and investigation, escalate to cases, and more.
 
-IMPORTANT: It is recommended to use {beats} to ship your data. Beat modules
-(for example, {filebeat-ref}/filebeat-modules.html[{filebeat} modules])
-are ECS-compliant so manual field mapping is not required, and all {es-sec-app}
-data is automatically populated. If you cannot use {beats}, map your data to
-{ecs-ref}[ECS fields] (see {ecs-ref}/ecs-converting.html[how to map data to ECS]).
-You can also try using the experimental https://github.com/elastic/ecs-mapper[ECS Mapper] tool.
+IMPORTANT: We recommend you use {beats} or {agent} integrations to ship your data to {es-sec}. Beat modules(for example, {filebeat-ref}/filebeat-modules.html[{filebeat} modules]) and {agent} integrations are ECS-compliant, which means data they ship to {es-sec} will automatically populate the relevant ECS fields.
+If you plan to use a custom implementation to map your data to {ecs-ref}[ECS fields] (see {ecs-ref}/ecs-converting.html[how to map data to ECS]), you should ensure that at least the "always required fields" below are populated, and ideally that all relevant ECS fields are populated. You can try using the experimental https://github.com/elastic/ecs-mapper[ECS Mapper tool] to create your custom implementation.
 
 [float]
-[[siem-host-fields]]
-== Host fields
-
-These fields must be mapped to display host data in the {es-sec-app}:
+[[siem-always-required-fields]]
+== Always required fields
+{es-sec} requires that all event and threat intelligence data be normalized to Elastic Common Schema.  For proper operation, all data must contain the following ECS fields:
```

Comment:
First tweak is just a flow/phrasing thing, take it or leave it. 😄 But for second tweak, common usage seems to be "_the_ Elastic Common Schema."
```suggestion
{es-sec} requires all event and threat intelligence data to be normalized to the Elastic Common Schema.  For proper operation, all data must contain the following ECS fields:
```